Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM cgr.dev/chainguard/go:latest as gobuild
```

Comment:
As @puffitos mentioned in https://github.com/caas-team/sparrow/pull/22#discussion_r1408883502

> Maybe use the go1.21 image equivalent, no need for latest :)

We should define the go version here

Reply:
since chainguards images by tag are not available for free I have switched to golang image for the app build.

Reply:
golang version will be passed to the dockerfile by --build-arg